### PR TITLE
Zoom with mouse wheel scroll

### PR DESCRIPTION
**Pull request by phweyland**
Change in that PR: zoom on mouse wheel scroll

### CODE REVIEW

File: src/gui/main.c [hunk 1]
```diff
@@ -84,7 +84,7 @@ char_callback(GLFWwindow* window, unsigned int c)
 static void
 scroll_callback(GLFWwindow *window, double xoff, double yoff)
 {
-  dt_gui_imgui_scrolled(window, xoff, yoff);
```
Comment (hanatos) [inline src/gui/main.c:87]: is this not called any more now? i suppose this causes trouble for std imgui widgets that interact via mouse wheel? does for instance light table still scroll down with this line removed?

i suppose we should just call both. there is a dt_gui_imgui_want_mouse() function to wrap imgui's mouse request, but it seems to be always on..

**Comment by hanatos** [comment]
nice! one question about the first bullet: did it not zoom centered around the mouse cursor before? you mean it centers the zoomed-in pixels under the mouse cursor instead of in the center of the screen?

File: src/gui/main.c (hunk 1 above)
Comment (phweyland) [reply src/gui/main.c:87]: > does for instance light table still scroll down with this line removed?

You are right. It doesn't work anymore.
Just added the line again.

**Comment by phweyland** [comment]
> did it not zoom centered around the mouse cursor before?

Just checked. It did actually. My brain ...

File: src/gui/main.c (hunk 1 above)
Comment (phweyland) [reply src/gui/main.c:87]: > i suppose we should just call both. there is a dt_gui_imgui_want_mouse() function to wrap imgui's mouse request, but it seems to be always on..

I haven't tested that (as it was commented in your code).

**Pull request merged**